internal/venice-common/src/main/java/com/linkedin/venice/pubsub/manager/TopicMetadataFetcher.java

```diff
@@ -62,7 +62,7 @@ class TopicMetadataFetcher implements Closeable {
   private static final int DEFAULT_MAX_RETRIES_FOR_POPULATING_TMD_CACHE_ENTRY = 5;
   private static final Duration INITIAL_RETRY_DELAY = Duration.ofMillis(100);
   private static final List<Class<? extends Throwable>> PUBSUB_RETRIABLE_FAILURES =
-      Arrays.asList(PubSubTopicDoesNotExistException.class, PubSubOpTimeoutException.class);
+      Collections.singletonList(Exception.class);
```

Comment:
This affects almost all of the APIs in this class. Is this on purpose?

Reply:
Yes thats on purpose and other call path could hit the same issue.